O365/account.py

```diff
@@ -109,6 +111,16 @@ def authenticate(self, *, scopes=None, **kwargs):
                 return False
 
         elif self.con.auth_flow_type == 'credentials':
+            if scopes is not None:
+                if self.con.scopes is not None:
+                    raise RuntimeError('The scopes must be set either at the Account instantiation or on the account.authenticate method.')
+                else:
+                    self.con.scopes = self.protocol.get_scopes_for(scopes)
+                    if '.default' not in scopes:
```

Comment:
Already said:

You can't rely on this. What if the user passed 'https://graph.microsoft.com/.default' to scopes?
Will then will add the .default scope twice.